libs/dataset-explorer/src/lib/ScatterUtils.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function getScatterColors(): string[] {
+  const colors = [];
+  for (let i = 0; i < 5; i++) {
+    colors.push("#6D8EF7", "#CA397E", "#F3B33E");

```

Comment:
I think these need to be consistent and come from the global FluentUI color palette here:
https://github.com/microsoft/responsible-ai-toolbox/blob/main/libs/core-ui/src/lib/util/FluentUIStyles.ts#L244
we want to make all of our charts consistent.
I tried to make all of our components as consistent as possible based on Ke's feedback but might have missed some.  We need to make all of our colors similar across charts.